tinker/static/assets/js/tinker.js

```diff
@@ -14,4 +14,117 @@ $(document).ready(function () {
             $(this).text("- Less Search Filters")
         }
     });
-});
\ No newline at end of file
+});
+
+function pagination() {
+    var numberOfItems = $(" #loop .list-group").length;
```

Comment:
seems like an odd thing to grab on? Maybe just add your own "pagination" class?

Reply:
Changed class name to `items-to-paginate`